pkg/fakedoc/generator.go

```diff
@@ -315,3 +327,40 @@ func (gen *Generator) loremIpsum(minlength, maxlength int, unit LoremUnit) strin
 		return lorem.Words(length)
 	}
 }
+
+func (gen *Generator) book(minlength, maxlength int, path string) (string, error) {
+	if minlength < 0 {
+		minlength = 0
+	}
+	if maxlength < 0 {
+		// FIXME: make bound on maximum length configurable
+		maxlength = minlength + 10
+	}
+
+	length := minlength + gen.Rand.IntN(maxlength-minlength)
```

Comment:
I think the random value should be in number of runes and not number of bytes.
The likelihood to split let's say Asian text at a an arbitrary byte index and resulting in an invalid string is very high.

Reply:
I did address this in the previous commit. Related: https://github.com/gocsaf/fakedoc/pull/5#discussion_r1869339086